.eslintrc

```diff
@@ -1,11 +1,16 @@
 {
   "parserOptions": {
-    "ecmaVersion": 2018
+    "ecmaVersion": 6
   },
-  "extends": ["eslint:recommended", "eslint-config-prettier"],
+  "extends": [
+    "eslint:recommended",
+    "eslint-config-prettier",
+    "plugin:@typescript-eslint/recommended"
+  ],
   "rules": {
     "no-unused-vars": "off",
-    "no-console": "off"
+    "no-console": "off",
+    "no-empty-function": "off"
```

Comment:
I suggest not adding this. If you REALLY want to allow this, you use `// eslint-disable-next-line no-empty-function`

Reply:
Okay, removed.